CONTRIBUTING.md

```diff
@@ -189,16 +191,11 @@ We follow [Google's Python Style Guide](https://google.github.io/styleguide/pygu
 for writing docstrings. Make sure your docstrings adhere to this style
 guide.
 
-The process for converting to a standard docstring format style is  
-ongoing. If you see a docstring in the code base that doesn't adhere
-to this formatting style and you'd like to contribute a fix, feel free
-to open a PR to correct the docstring formatting.
-
 ###### Code Style
 
-We use [pylint](https://pypi.org/project/pylint/),
-[black](https://black.readthedocs.io/en/stable/the_black_code_style/index.html),
-and [ruff](https://github.com/astral-sh/ruff) in our CI via
+We use [pylint](https://pypi.org/project/pylint/), [prettier](https://prettier.io/),
```

Comment:
```suggestion
We use [prettier](https://prettier.io/),
```
I think we got rid of Pylint in the last quarter🙂

Reply:
ahhhh that's right! Removing!